frontend/src/metabase/common/components/Input/Input.styled.tsx

```diff
@@ -138,6 +138,13 @@ export const InputLeftButton = styled(InputButton)<InputButtonProps>`
   left: 0;
 `;
 
+export const InputLeftSection = styled.div`
```

Comment:
Ideally, we wouldn't be building on this component. We should leverage the Mantine Input components